spec/support/shared_examples/crud.rb

```diff
@@ -128,6 +128,21 @@
         end
       end
     end
+
+    if methods.include? :search
+      context '::search' do
+        let(:status) { 200 }
+        let(:response) { { described_class.plural_key => [{}] } }
+        let(:params) { property ? { property.to_sym => '13666', first_name: "John" } : {} }
+
+        it { expect(described_class.search(params)).to be_a(DaisybillApi::Ext::PageableCollection) }
+        it { expect(described_class.search(params)).to have(1).records }
+        it { expect(described_class.search(params)).to all(be_a described_class) }
+        if prefix
+          it { expect(described_class.search(params).map(&:"#{property}")).to all( eq 13666 ) }
```

Comment:
Line is too long. [95/80]<br>Separate every 3 digits in the integer portion of a number with underscores(_).<br>Space inside parentheses detected.